### PR TITLE
Update 02-native.md

### PR DESCRIPTION
**Pull request by antongisli**
pixie is the phonetic way to say PXE - replacing it with the correct spelling.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/970#" title="Author self-approved">antongisli</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @antongisli. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/03-edge/02-native.md [hunk 1]
```diff
@@ -103,32 +103,32 @@ needs. Detailed instructions for this are provided below.
...
- config: 
-   # disable the built in cni
-   flannel-backend: none
-   no-flannel: true
-   disable-network-policy: true
-   Disable:
-     - metrics-server
-```
-
+If the edge location configuration is known and predictable Staging, Installation, and Registration can be potentially combined into one step by the central IT/Ops team and then ship the fully configured edge hosts to the edge location. The Site Operator at the edge location needs to hook up power and network cable without any further configuration. The edge cluster will be ready to be
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/02-native.md:106]:
```suggestion
If the edge location configuration is known and predictable, then Staging, Installation, and Registration can be combined into one step by the central IT/Ops team, and then ship the fully configured edge hosts to the edge location. The Site Operator at the edge location needs to hook up power and network cable without any further configuration. The edge cluster will be ready to be 
```
Minor cleanup.

File: content/docs/04-clusters/03-edge/02-native.md [hunk 2]
```diff
@@ -209,7 +209,7 @@ stages:
 For bare metal edge hosts, creating the palette edge installer variant involves generating an installer image which is derived by customizing the default Palette Edge Installer. Site-specific settings described above are baked into this image. The customization is performed by using the Palette Edge Installer Container. 
 
 Following are the steps to customize **site settings** and **build an installer image**:
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/02-native.md:211]:
```suggestion
The following are steps to customize *site settings* and build an *installer image*: 
```
Bold is used for UI components.

File: content/docs/04-clusters/03-edge/02-native.md [hunk 3]
```diff
@@ -209,7 +209,7 @@ stages:
 For bare metal edge hosts, creating the palette edge installer variant involves generating an installer image which is derived by customizing the default Palette Edge Installer. Site-specific settings described above are baked into this image. The customization is performed by using the Palette Edge Installer Container. 
 
 Following are the steps to customize **site settings** and **build an installer image**: 
-
+
 <br />
 
 1. Checkout the following [Git Repo](https://github.com/spectrocloud/pxke-samples) on your local machine or server where you intend to run the customization procedure.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/02-native.md:215]:
```suggestion
1. Checkout the following [Git Repo](https://github.com/spectrocloud/pxke-samples) on your local machine or server where you intend to invoke the customization procedure.
```
Avoid ableism. Replacing run with invoke.

**Review by karl-cardenas-coding — APPROVED** [review]

**Pull request merged**